src/ui/public/timepicker/relative_panel/kbn_timepicker_relative_panel.html

```diff
@@ -0,0 +1,139 @@
+<form role="form" ng-submit="applyRelative()" class="form-inline" name="relativeTime">
+  <div class="kbn-timepicker-body kuiVerticalRhythm">
+    <div class="kbn-timepicker-section kbn-timepicker-body-column">
+      <div class="kuiLocalDropdownHeader kbn-timepicker-form-header">
+        <label class="kuiLocalDropdownHeader__label">
+          From
+        </label>
+
+        <div class="kuiLocalDropdownHeader__actions">
+          <a
+            class="kuiLocalDropdownHeader__action"
+            ng-click="setRelativeToNow({key:'from'})"
+            kbn-accessible-click
+          >
+            Set To Now
+          </a>
+        </div>
+      </div>
+
+      <div class="kuiLocalDropdownFormNote kuiVerticalRhythmSmall">
+        <span ng-show="relative.from.preview">
+          {{relative.from.preview}}
+        </span>
+        <span ng-hide="relative.from.preview">
+          <em>Invalid Expression</em>
+        </span>
+      </div>
+
+      <div class="kuiFieldGroup kuiVerticalRhythmSmall">
+        <div class="kuiFieldGroupSection kuiFieldGroupSection--wide" ng-class="{ 'has-error': checkRelative() }">
```

Comment:
Instead of the `ng-class` here, can we put it on the input instead?

```html
<input
  ng-class="{ 'kuiTextInput-isInvalid' : checkRelative() }"
  etc...
>
```

You will need to rebase master to make this class available though, since I just added it.